x-pack/plugin/security/src/main/bin/elasticsearch-reset-kibana-system-password

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0; you may not use this file except in compliance with the Elastic License
+# 2.0.
+
+ES_MAIN_CLASS=org.elasticsearch.xpack.security.authc.esnative.tool.ResetBuiltinPasswordTool \
```

Comment:
An additional CLI tool is what made most sense after our discussion today. In the long run we can re-consider the possibility of a single tool for all builtin users ( replacing setup-passwords ) but for now I presume this has small added value.

Reply:
That was my feeling as well - we should have a conversation about a generic "reset-user-password" tool. 
But there are questions:
- is it "reset many users at once" ?
- is it only reserved users, or native users as well?
- does it have a batch mode for scripting?

And today is not the day to decide on those.